canonical-web-and-design/ubuntu.com
# Display UA subscriptions in the list.

## Done

- Add selector for UA subscriptions.
- Display UA subscriptions in the subscription list.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10290.demos.haus/advantage?test_backend=true).
- You should see all your UA subscriptions in the list on the left hand side.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/155.

## Screenshots

<img width="457" alt="Screen Shot 2021-09-03 at 10 36 44 am" src="https://user-images.githubusercontent.com/361637/131933092-d601fcc1-0542-4e09-8a35-a2ee6dc03078.png">
